controllers/limitador_controller.go

```diff
@@ -116,9 +119,47 @@ func (r *LimitadorReconciler) reconcileSpec(ctx context.Context, limitadorObj *l
 		return ctrl.Result{}, err
 	}
 
+	if err := r.reconcilePdb(ctx, limitadorObj); err != nil {
+		return ctrl.Result{}, err
+	}
+
 	return ctrl.Result{}, nil
 }
 
+func (r *LimitadorReconciler) reconcilePdb(ctx context.Context, limitadorObj *limitadorv1alpha1.Limitador) error {
+	if limitadorObj.Spec.PodDisruptionBudget == nil {
+		return nil
+	}
+
+	limitadorPdb := limitadorObj.Spec.PodDisruptionBudget
+	if err := limitador.ValidatePDB(limitadorPdb); err != nil {
+		return err
+	}
+
+	limitadorPdb.Selector = &metav1.LabelSelector{
+		MatchLabels: map[string]string{"app": "limitador"},
```

Comment:
A better select is going to be needed. You can create two limitador CRs in the same name space with different pod disruption budgets and both would have the `app: limitador` labels. In the pod labels there is a `pod-template-hash` which could be used, but I am not sure how we would get that value. If not we may start adding unique labels to all the resources related to a limitador CR. This might not be a bad idea but would be somewhat more work (maybe).

Reply:
This is another thing we considered. Your assumption is true that two limitador CR's in the same namespace would result in the selectors matching both. However we noticed that the selector for the `Service` is also matching these exact (non-unique) labels https://github.com/Kuadrant/limitador-operator/blob/main/pkg/limitador/k8s_objects.go#L50. This made me assume that two limitador CR's in the same namespace was either not intended or unsupported.

If this assumption is incorrect we need to update the labels on the deployment (could simply use a new label with the limitador CR name).

Reply:
In this case of using the same labels, do you any thoughts on exporting that `lables()` function and using it here instead of manually defining the same labels again ? 🤔

Reply:
> In this case of using the same labels, do you any thoughts on exporting that `lables()` function 

Sure, I saw at least one other case of duplication so could update both. I'll hold off on this change though until we decide whether we should be supporting multiple limitador CR's in a namespace or not

Reply:
Just pushed a change now that adds a new unique label following the same naming semantics as in  https://github.com/Kuadrant/authorino-operator/blob/main/pkg/resources/k8s_util.go#L16. This should solve the multiple limitador CR's in a single namespace issue.